samples/echoWebhookBot/go.mod

```diff
@@ -2,6 +2,6 @@ module github.com/PaulSonOfLars/gotgbot/samples/echoWebhookBot
 
 go 1.15
 
-require github.com/PaulSonOfLars/gotgbot/v2 v2.0.0-rc.8
+require github.com/PaulSonOfLars/gotgbot/v2 v2.99.99
```

Comment:
the `replace` takes whatever version is checked out in the parent directory, so I'd put a fake version here to avoid confusion.

Reply:
Even if it doesn't change anything, setting this to the minimum acceptable version compatible with the program could be a good practice. If the sample program don't use a changed API, we don't update this.

Also, to work with multiple go.mod in a single repository: https://go.dev/doc/tutorial/workspaces

Reply:
Yeah, fair.

Re: workspaces; I'm aware of them, I had simply opted to use the require here because i wanted to point out that this isnt intended to be a working go.mod file. If I understand it right, even if we used a workspace, we would still have to use the dummy versions you mentioned here, so wouldn't really solve much :)